e2e-tests/disable-api-if-not-fully-synced-flag.sh

```diff
@@ -104,6 +104,18 @@ if ! [[ $MSG = *"Canister state is not fully synced."* ]]; then
   exit 1
 fi
 
+# bitcoin_get_block_headers should panic.
```

Comment:
I believe it will not be a problem if it stays. AFAIK we agreed not to hide the method behind the feature flag, and just left the endpoint exposed, furthermore I will add the rest of the e2e test very soon, I believe there will be no releases prior to that, hence I think it will be time wasteful to remove this now, and to put it back later.

Reply:
My understanding was that, at least so far, we're not adding the endpoint until Manu confirms. We can wait until next week when he's back if you like.

Reply:
Can you please PTAL, I returned unimplemented whenever the endpoint is called. Alternatively, we can remove(or comment) it from canister/main.rs, but I believe this is enough.